README.md

```diff
@@ -3,10 +3,78 @@
 ![build](https://github.com/endjin/dependabot-pr-parser/workflows/build/badge.svg)
 [![GitHub license](https://img.shields.io/badge/License-Apache%202-blue.svg)](https://raw.githubusercontent.com/corvus-dotnet/Corvus.Deployment/master/LICENSE)
 
+This repo contains 3 docker-based GitHub actions that can be used to enable continuous integration for a configurable subset of dependencies being managed by GitHub Dependabot.
+
+## Overview
```

Comment:
Is it worth adding a section that points to projects that are making use of this, and thus you can show people what common json configuration files look like?